processor/blockprocessor/exporter_processor.go

```diff
@@ -0,0 +1,111 @@
+package blockprocessor
+
+import (
+	"fmt"
+	"github.com/algorand/go-algorand/agreement"
+	"github.com/algorand/go-algorand/config"
+	"github.com/algorand/go-algorand/data/bookkeeping"
+	"github.com/algorand/go-algorand/ledger"
+	"github.com/algorand/go-algorand/ledger/ledgercore"
+	"github.com/algorand/go-algorand/rpcs"
+	"github.com/algorand/indexer/data"
+	"github.com/algorand/indexer/exporters"
+	indexerledger "github.com/algorand/indexer/processor/eval"
+	"github.com/sirupsen/logrus"
+)
+
+type ledgerExporterProcessor struct {
+	exporter exporters.Exporter
+	ledger   *ledger.Ledger
+	logger   *logrus.Logger
+}
+
+func (proc *ledgerExporterProcessor) AddGenesisBlock() error {
+	if proc.exporter != nil && uint64(proc.ledger.Latest()) == 0 {
+		blk, err := proc.ledger.Block(0)
+		if err != nil {
+			return fmt.Errorf("addGenesisBlock() err: %w", err)
+		}
+		err = proc.exporter.Receive(data.BlockData{
+			BlockHeader: blk.BlockHeader,
+			Payset:      blk.Payset,
+			Delta:       &ledgercore.StateDelta{},
+			Certificate: &agreement.Certificate{},
+		})
+		if err != nil {
+			return fmt.Errorf("addGenesisBlock() handler err: %w", err)
+		}
+	}
+	return nil
+}
+
+// Process a raw algod block
+func (proc *ledgerExporterProcessor) Process(blockCert *rpcs.EncodedBlockCert) error {
```

Comment:
it looks like the same logic is already ran in [block_processor,](https://github.com/algorand/indexer/blob/45d5c4b28523098696f438d98602ad411f84832e/processors/blockprocessor/block_processor.go#L127) why do we need this in the exporter again?